liblangutil/Exceptions.h

```diff
@@ -40,6 +40,8 @@ struct CompilerError: virtual dev::Exception {};
 struct InternalCompilerError: virtual dev::Exception {};
 struct FatalError: virtual dev::Exception {};
 struct UnimplementedFeatureError: virtual dev::Exception {};
+struct InvalidAstError: virtual dev::Exception {};
```

Comment:
Why langutil and not libsolidity? Isn't ast specific to solidity?

Reply:
yes, but there was no Exceptions.h in libsolidity so i thought it might be simpler to just add it here instead of creating a new file just for that one exception. Should I? Or where else does it make more sense?

Reply:
Ah ok, since the other exceptions are also all solidity-specific (even if not in their name), it's probably better to keep it that way.